docs/manage-metadata-using-gravitino.md

```diff
@@ -733,6 +733,70 @@ In addition to the basic settings, Gravitino supports the following features:
+| HASH            | `hash`  | `Strategy.HASH`  | Bucket table using hash                                                                     |
+| RANGE           | `range` | `Strategy.RANGE` | Bucket table using range                                                                    |
+| EVEN            | `even`  | `Strategy.EVEN`  | Bucket table using even, The data will be bucketed equally according to the amount of data. |
+
+- Number. It defines how many buckets we use to bucket the table.
+- Function arguments. It defines which field or function should be used to bucket the table. Gravitino supports the following three kinds of arguments, for more, you can refer to Java class `FunctionArg` and `DistributionDTO` to use more complex function arguments.
+
+| Expression type | Json example                                                      | Java example                                                                                            | Equivalent SQL semantics | Description                    | 
+|-----------------|-------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|--------------------------|--------------------------------|
+| Field           | `{"type":"field","fieldName":["score"]}`                          | `FieldReferenceDTO.of("score")`                                                                         | `score`                  | field reference value `score`  |
+| Function        | `{"type":"function","functionName":"hour","fieldName":["score"]}` | `new FuncExpressionDTO.Builder()<br/>.withFunctionName("hour")<br/>.withFunctionArgs("score").build()`  | `hour(score)`            | function value `hour(score)`   |
+| Constant        | `{"type":"constant","value":10, "dataType": "integer"}`           | `new LiteralDTO.Builder()<br/>.withValue("10")<br/>.withDataType(Types.IntegerType.get())<br/>.build()` | `10`                     | Integer constant `10`          |
+
+
+#### Sorted order table
+
+To define a sorted order table, you should use the following three components to construct a valid sorted order table.
+
+- Direction.  It defines in which direction we sort the table.
+
+| Direction  | Json   | Java                       | Description                               |
+| ---------- | ------ | -------------------------- |-------------------------------------------|
+| Ascending  | `asc`  | `SortDirection.ASCENDING`  | Sorted by a field or a function ascending |
+| Descending | `desc` | `SortDirection.DESCENDING` | Sorted by a field or a function ascending |
+
+- Null ordering. It describes how to handle null value when ordering
+
+| Null ordering                     | Json          | Java                       | Description                       |
+| --------------------------------- | ------------- | -------------------------- |-----------------------------------|
```

Comment:
Why do use description in the first column here?

Reply:
No, it's in the last column.